storageservice/modules/files/file_client.bal

```diff
@@ -78,7 +78,7 @@ public isolated client class FileClient {
         }
         map<string> headerMap = populateHeaderMapFromRequest(request);
         http:Response response = <http:Response>check self.httpClient->get(requestPath, headerMap);
-        if (response.statusCode === http:STATUS_OK ) {
+        if (response.statusCode === http:STATUS_OK) {
```

Comment:
what is the reason to use `===`? We have `==` also in Ballerina right?

Reply:
This connector was initially implemented using older ballerina swan lake versions. There are places that we can optimize and improve with the latest version. I will work on this.